Add March 2025 newsletter (#2335)
writethedocs/www: docs/blog/newsletter-march-2025.rst

Several mentioned that when communicating by text, they’re more focused and edit their text, while in person they may not be as focused and tend to "ramble". One issue is personal processing style. Some people prefer to communicate by text (async), which gives them time to process information. Others prefer the spontaneity of synchronous conversations. 

Others noted that they rely on vocal tone and body language when communicating in person. Sometimes emojis can relay that in text, but there are concerns about whether emojis get interpreted as intended. Alternatively, some people find it hard to evaluate the tone of text if emojis aren’t used: “Is this content passive aggressive or dead serious or what?”

Is it safe or proper to express one’s personality, sense of humor, and personal, non-work opinions at work? Some have a “what you see [or read] is what you get” attitude. Some indicate that they are more conscious of word choice and limit personal sharing when interacting at work -- unless they’re with work friends.

> For those people who primarily work online, some expressed concerns about too many text messages and notifications or excessive use of emojis. They caution others to be sensitive to over texting: writing too much or too often. 

Reviewer: Does "over texting" need a hyphen?

Author: ```suggestion
For those people who primarily work online, some expressed concerns about too many text messages and notifications or excessive use of emojis. They caution others to be sensitive to over-texting: writing too much or too often. 
```

> See more Write the Docs resources about `working across roles </topics/#working-across-roles>`__.

Reviewer: This seems like it might be in the wrong place. Is it supposed to be after the stakeholders item below?

Reviewer: Oh, I see - yes it's also below the stakeholders item here: https://github.com/writethedocs/www/pull/2335/files#diff-59b3581b4a6273705593a1664bb1976cec5ed10a247c6c660d0043835eeb2965R48 - I'm guessing it got duplicated accidentally?

Author: The other one is working _with other_ roles. This one is working _across_ roles. They have similar meanings, but the arrangements in the topics seemed slightly different. I figured we might rearrange the whole thing soon anyway so I didn't want to worry too much about it. 🤷 Open to other suggestions though for sure!

-----------------------------------------
Contending with unresponsive stakeholders
-----------------------------------------

> When stakeholders like subject matter experts (SMEs) don't provide feedback, it can be difficult or impossible to stick to project timelines and meet your goals. To encourage accountability on docs projects, consider a policy that directly addresses what will happen when you can't get feedback form stakeholders.

Reviewer: ```suggestion
When stakeholders like subject matter experts (SMEs) don't provide feedback, it can be difficult or impossible to stick to project timelines and meet your goals. To encourage accountability on docs projects, consider a policy that directly addresses what will happen when you can't get feedback from stakeholders.
```

See more Write the Docs resources about `working with other roles </topics/#working-with-other-roles>`__.

---------------------------------------
Maximizing the value of weekly meetings
---------------------------------------

Effective meetings are essential for keeping documentation teams aligned, engaged, and productive. However, balancing efficiency with meaningful interaction can be challenging, especially for remote and distributed teams. To make the most of your meeting time, consider these practical strategies for fostering engagement, boosting productivity, and strengthening team cohesion.

- Maintain a consistent meeting cadence.

  - Keep a regular meeting schedule to foster team connection, particularly in remote environments.
  - Use meetings to check in, share updates, and reinforce team rapport.

>   - Even if there are no pressing topics, use the time for casual catch-ups or knowledge-sharing.

Author: ```suggestion
  - Even if there are no pressing topics, use the time for casual catch-ups or knowledge sharing.
```

- Introduce special topics and deep dives.

  - Alternate between standard updates and in-depth discussions.
  - Explore broader topics such as SEO strategies, competitor documentation analysis, or industry trends for professional development.
  - Encourage team members to lead discussions on topics relevant to their expertise.

- Incorporate structure and social elements.

  - Follow a structured agenda with backlog reviews, prioritization, and resolution of blocking issues.
  - Add social elements such as emoji-based temperature checks, show-and-share sessions, or kudos time to keep engagement high.

> - Leverage retrospectives and knowledge sharing.

Reviewer: We hyphenated "knowledge sharing" here: https://github.com/writethedocs/www/pull/2335/files#diff-59b3581b4a6273705593a1664bb1976cec5ed10a247c6c660d0043835eeb2965R60

  - Allow for casual discussions or early dismissals when topics are light to maximize time management.

By implementing alternating formats, structured agendas, and social engagement strategies, you can transform routine meetings into valuable, collaborative touchpoints that enhance productivity and team cohesion.

See more Write the Docs resources about `planning and how we work </topics/#planning-and-how-we-work>`__.

-----------------------------------------------
Where to keep content for reuse in docs-as-code
-----------------------------------------------

A recent question about `#docs-as-code <https://writethedocs.slack.com/archives/C72NZ18FR>`__ focused on whether to store content that would be available for reuse (through `transclusion <https://en.wikipedia.org/wiki/Transclusion>`__) together with the rest of your content (colocation) or in a separate directory (such as `_partials`). The asker felt strongly that a separate location was best, but was getting some pushback that developers not focused on writing would prefer colocation.

> The arguments for a separate location included a greater sense of order, relief from fear that the snippets to include would be built as complete pages, and the use of import aliases to not have to worry about relative paths. One person offered the analogy of utility functions in code, which almost no one would advocate for colocating with code if they're reused in various files.

Reviewer: In the first sentence, something about the phrasing in that third element in the series seems not-quite-parallel to me. The first two things are phrased to emphasize a benefit. Can the third be revised to something like "...and not having to worry about relative paths when using import aliases" to emphasize the benefit without changing the meaning? Maybe I'm overthinking this.

Author: ```suggestion
The arguments for a separate location included a greater sense of order, relief from fear that the snippets to include would be built as complete pages, and not having to worry about relative paths through the use of import aliases. One person offered the analogy of utility functions in code, which almost no one would advocate for colocating with code if they're reused in various files.
```

  :align: center
  :width: 50%
  :target: https://mintlify.link/ZZO1Kan
  :alt: Mintlify logo

`Mintlify <https://mintlify.link/ZZO1Kan>`__ helps you create beautiful documentation that converts users.

Documentation is as important as the product, so we help your docs look polished and stay up to date without having to bug your engineers. From AI-powered writing assistance to Git syncing for better collaboration, Mintlify takes care of the technical details so you can focus on writing quality content.

We’re passionate about up-leveling docs across the board, so we created a `technical writing guide <https://mintlify.link/4ayAHSQ>`__ for non-writers—give it a read and let us know if you have any suggestions!

*Interested in sponsoring the newsletter? Take a look at our* `sponsorship prospectus </sponsorship/newsletter/>`__.

------------------
Featured job posts
------------------

`Technical Copywriter <https://www.linkedin.com/jobs/view/4164570544>`__, RLE Technologies - *Remote*

*Interested in promoting your open position? See our* `job posting sponsorship </sponsorship/jobs/>`__ *for more details.*

------------------------
Write the Docs resources
------------------------

> Write the Docs offers lots of valuable resources related to documentation. See all of the Write the Docs `learning resources </about/learning-resources/>`__. To discuss any of these ideas or others related to documentation, join the conversation in the `Write the Docs Slack community </slack/>` in one of the many `channels </slack/#channel-guide>`__.

Reviewer 2: ```suggestion
Write the Docs offers lots of valuable resources related to documentation. See all of the Write the Docs `learning resources </about/learning-resources/>`__. To discuss any of these ideas or others related to documentation, join the conversation in the `Write the Docs Slack community </slack/>`_ in one of the many `channels </slack/#channel-guide>`__.
```
